test/selenium/test_options.py

```diff
@@ -11,13 +12,13 @@ def test_options(self):
     def test_show_counter(self):
         selector = '#showCounter'
         self.load_options()
+        sleep(3)
+
         el = self.query_selector(selector)
         self.assertTrue(el.is_selected())
         el.click()
 
         self.driver.refresh()
-
-        self.load_options()
```

Comment:
I think we want to keep this, since we want to assure that the option is selected *after the options page is reloaded*, not just that it's checked on the same page after we check it.

Reply:
We'll need to add an additional sleep after this too, I suspect.

Reply:
I'm not sure if it's necessary since the tests are running one by one. Otherwise do we need another load_option after the second reload?